### ISSUE TITLE
Sync with upstream

### ISSUE THREAD

**Issue by mchugh19** [wcannon/salt-check#21]
Upstream salt module is getting updates for the utils, and unicode changes in 2018. But there are updates in this repo that have not yet been pushed upstream, making support awkward. Can you please sync with upstream and submit updates?

**Comment by wcannon** [comment]
Good point.  I'll work on getting the updates from salt open into this repo this week.

**Comment by wcannon** [comment]
I think I misunderstood your request.  Are you wanting the updates in this repo to get pushed back into salt open?

**Comment by mchugh19** [comment]
Yeah. There have been some unicode updates in saltopen, while you have some fixes in your branch. So we'll need to have this branch merged with saltopen, and then all the fixes submitted back to open.
